pytools/__init__.py

```diff
@@ -1510,6 +1512,28 @@ def __str__(self):
 
         return "\n".join(lines)
 
+    def github_markdown(self):
+        columns = len(self.rows[0])
+        col_widths = [max(len(row[i]) for row in self.rows)
+                      for i in range(columns)]
+
+        alignments = self.alignments
+        # If not all alignments were specified, extend alignments with the
+        # last alignment specified:
+        alignments += self.alignments[-1] * (columns - len(self.alignments))
+
+        lines = [" | ".join([cell.center(col_width) if align == 'c'
+            else cell.ljust(col_width) if align == 'l' else cell.rjust(col_width)
```

Comment:
Get all the `if`s to the beginning of the line.

Reply:
Since the ifs bind to the code before them in this case (e.g., `cell.center(col_width) if align == 'c'` belongs together) , doesn't it make sense to keep them on the same line?

Reply:
Fair enough.